Src/ParachutesControl.c

```diff
@@ -45,17 +68,94 @@ int32_t readPressure(BarometerData* data)
     return (int32_t)pressure;
 }
 
-void filterSensors(int32_t current_accel, int32_t current_pressure, int32_t positionVector[3])
+
+
+/**
+ * Takes an old state vector and current state measurements and
+ * converts them into a prediction of the rocket's current state.
+ *
+ * Params:
+ *   oldState - (KalmanStateVector) Past altitude, velocity and acceleration
+ *   currentAccel - (double) Measured acceleration
+ *   currentAltitude - (double) Measured altitude
+ *   dt - (double) Time since last step
```

Comment:
Hey is this in milliseconds? That's what it's coming in as.

Reply:
Thank you for reminding me. I'll convert this too.